src/constants/color.js

```diff
@@ -1,6 +1,11 @@
 // @ts-check
 
-export const GRAVITY = { x: 0, y: 0.1 }
 export const RED = 'rgb(212, 42, 32)'
 export const YELLOW = 'rgb(250, 194, 43)'
 export const BLUE = 'rgb(14, 99, 142)'
+
+export const Color = Object.freeze({
```

Comment:
A frozen object just means that you can't add or change properties. But it's "shallow" freeze - nested objects are not recursively frozen.

It's useful for enums. We want to access properties on these objects but never change them.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze